pages/docs/ttn-sensor.mdx

```diff
@@ -31,25 +30,25 @@ Dort angekommen, kannst du einen neuen Webook für deine Applikation einrichten.
 
 ### Webhook Parameter
 
-Die **Webhook ID**, respektive der Name deines Webhooks, ist frei wählbar und ganz dir überlassen. Das **Webhook format** ist JSON.
+Die **Webhook ID**, respektive der Name deines Webhooks, ist frei wählbar und ganz dir überlassen. Das **Webhook-Format** ist *JSON*.
 
-&#x1f4a1; Was beim TTN V3 Stack der Webhook ist, war beim TTN V2 Stack die HTTP-Integration. Die Funktionsweise ist die gleiche: sowohl die HTTP-Integration als auch der Webhook können *POST* und *GET* Operationen über HTTP an einem bestimmten Endpoint ausführen.
+&#x1f4a1; *Was beim TTN V3 Stack der Webhook ist, war beim TTN V2 Stack die HTTP-Integration. Die Funktionsweise ist die gleiche: sowohl die HTTP-Integration als auch der Webhook können *POST* und *GET* Operationen über HTTP an einem bestimmten Endpoint ausführen.*
 
-Unter den Endpoint Settings muss nun der API URL konfiguriert werden. In unserem Falle nutzen wir als **Base URL** des Endpoints die RESTful API von Stadtpuls: 
+Unter den Endpoint-Settings muss nun die API URL konfiguriert werden. In unserem Falle nutzen wir als **Base URL** des Endpoints die RESTful API von Stadtpuls: 
 
 ```plain
 https://api.stadtpuls.com
 ```
 
-Wir übergeben der API jedoch noch eine zusätzliche Information – **Additional headers** mit dem Namen *authorization* im Header: nämlich deinen Authorizierungs-Token den du dir bereits angelegt hast.
+Wir übergeben der API jedoch noch eine zusätzliche Information – **Additional headers** mit dem Namen *authorization* im Header: nämlich deinen Authorisierungs-Token, den du dir bereits angelegt hast.
```

Comment:
Ich glaube es müsste eigentlich "Authentifizierungs-Token" sein https://translate.google.com/?sl=auto&tl=de&text=HTTP%20authentication&op=translate

Reply:
agree to both. Please go ahead and change it according to your suggestion. Good find!